src/components/FeedbackBox/FeedbackBox.stories.tsx

```diff
@@ -1,6 +1,7 @@
 import { Box } from "@chakra-ui/react";
 import type { Meta, StoryObj } from "@storybook/react";
 import { useState } from "react";
+import { userEvent, expect, screen } from "@storybook/test";
```

Comment:
Did not know you can get the mounted DOM directly from the package. I've been doing `const screen = within(canvasElement);`

Reply:
Right, something to remember for `Modal` for sure bc `screen` here is not the same as `within(canvasElement)`